src/gui/search/searchsortmodel.h

```diff
@@ -30,10 +30,12 @@
 #define SEARCHSORTMODEL_H
 
 #include <QSortFilterProxyModel>
+#include <QStringList>
 #include "base/utils/string.h"
 
 class SearchSortModel: public QSortFilterProxyModel
 {
+    using base = QSortFilterProxyModel;
```

Comment:
> I've seen this somewhere... C# or Java (or something like that). I won't focus on this.

This keyword was proposed for C++ too, and during discussion what to do with the multiple inheritance somebody came up with such a `typedef`, which solved the problem.

> ...unless new base class has the same functions :)

Yes, of course it has to have the same functions with the same signatures. Qt's model classes are a good example of such base classes.